rook/main.py

```diff
@@ -32,6 +32,16 @@
 
 warnings.simplefilter('default', DeprecationWarning)
 
+# set up colors
+# TODO add feature to turn coloring on and off
+norm_color = '\033[0m'  #reset color
+skip_color = '\033[90m' #dark grey
+fail_color = '\033[91m' #red
+pass_color = '\033[92m' #green
+name_color = '\033[93m' #yellow
+time_color = '\033[94m' #blue
```

Comment:
in Windows this "colors" are not rendered and the results of the screen output is quite ugly. Is there a way to make this cross-platform?

Reply:
Maybe? Some small effort could make using color togglable; maybe that would be a good entry-level improvement from one of the soon-to-start team?

Reply:
see #985